tests/unit_tests/test_wave_06.py

```diff
@@ -32,13 +32,10 @@ def test_get_no_matching_items_by_category():
     )
 
     items = vendor.get_by_category("Electronics")
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-
-@pytest.mark.skip
+    assert len(items) == 0
+    assert items == []
```

Comment:
These two asserts are testing for a very similar outcome. I would argue that only one over the other is needed! Specifically, I would check for the empty list as that covers your other assert as well!